packages/xchain-ethereum/src/client.ts

```diff
@@ -669,15 +692,13 @@ export default class Client implements XChainClient, EthereumClient {
 
         estimate = await contract.estimateGas.transfer(recipient, txAmount, {
           from: this.getAddress(),
-          gasPrice: gasPriceBN,
```

Comment:
Why don't we need a value for `gasPrice` anymore?

Reply:
We don't need gasPrice for gas limit estimation.